google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -45,6 +45,10 @@ appengine.appyaml.location.label=app.yaml path\:
 appengine.dockerfile.location.browse.button=Browse for Your Dockerfile Location
 appengine.deployment.error.with.code=Deployment failed with exit code: {0}\nPlease make sure that you are using the latest version of the Google Cloud SDK.\nRun ''gcloud components update'' to update the SDK. (See: https://cloud.google.com/sdk/gcloud/reference/components/update.)
 appengine.deployment.error.during.execution=Deployment failed due to an unexpected error during the deployment execution phase.
+appengine.stop.modules.version.execution.error=Failed to stop application due to an unexpected error during the execution phase.
+appengine.stop.modules.version.execution.error.with.code=Stop application failed with exit code: {0}
+appengine.stop.modules.version.success.dialog.message=Application successfully undeployed.
```

Comment:
I guess the correct terminology would be "stopped", not "undeployed". Also, maybe specify which application in the message?
"App Engine application {0} has successfully stopped." or something like that.